boards/m5stack/m5stack_cores3/Kconfig.defconfig

```diff
@@ -9,6 +9,6 @@ config INPUT_FT5336_INTERRUPT
 	default y if INPUT
 
 config INPUT
-	default y
+	default y if LVGL || DISPLAY
```

Comment:
Should this be also addressed for `m5stack_core2` board?

Reply:
On CoreS3/CoreS3SE, the I2C-connected GPIO Expander AW9523B is connected to various parts of the system, so it should be enabled at all times, but this enables I2C, which causes a conflict with the mcuboot settings.
On Core2, there is no GPIO Expander, so no problems occur.